core/src/main/java/com/amplifyframework/core/model/types/JavaFieldType.java

```diff
@@ -50,14 +53,19 @@
     STRING(String.class.getSimpleName()),
 
     /**
-     * Represents the java.util.Date data type.
+     * Represents the Date data type.
```

Comment:
As written, a customer might get confused thinking `Date` is `java.util.Date`.    This should be `AWSDate` (or whatever the model class is renamed, if we are renaming it).  

If we do rename `AWSDate`, I don't like just dropping the `AWS` prefix, since `Date` would be confused with `java.util.Date`.  Maybe `AmplifyDate`?  Not sure about that either though.

Reply:
We have a big debate in iOS too since this is consumer-facing API. I'm honestly against this verbosity of adding AWS or Amplify as prefixes (or any other prefix tbh) not only to the verbosity itself, but why is it relevant that is an **Amplify** Date? From a consumer perspective that makes me wonder why that's a special Date/DateTime.

I really wanted customer to be able to ready is and understand what the intent is: it's a `Date`, a `DateTime` and `Time`.

But I'm starting to accept that I will lose this battle.

Reply:
Just throwing this out there, one other option for naming would be:

AWSDate > com.amplifyframework.core.model.OffsetDate
AWSDateTime > com.amplifyframework.core.model.OffsetDateTime
AWSTime > com.amplifyframework.core.model.OffsetTime
AWSTimestamp > java.util.Date

These are basically the same names that are used by java.time, but we've sort of reimplemented them ourselves, since java.time isn't available.

@jamesonwilliams any thoughts?